msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -803,25 +803,136 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
+|`tabs`| array | |✔️| Configures custom tabs on the Office application ribbon. |
+|`tabs.id`| string | 64 characters | | Unique identifier for this tab within the app.|
+|`tabs.label`| string | 64 characters | | Displayed text for the tab.|
+|`tabs.position`| object | | | Configures the position of the custom tab relative to other tabs on the ribbon.|
+|`tabs.position.builtinTabId`| string | 64 characters | | The ID of the built-in tab. See [Find the IDs of controls and control groups](/office/dev/add-ins/design/built-in-button-integration#find-the-ids-of-controls-and-control-groups).|
+|`tabs.position.align`| string enum | | |  Defines alignment of this custom tab relative to the specified built-in tab. Supported values: `after`, `before`|
+|`tabs.groups`| string |64 characters | | Defines tab groups.|
+|`tabs.groups.id`| string |64 characters | | Unique identifier for this tab group within the app. Must be different from any built-in group id in the Office application and any other custom group.|
+|`tabs.groups.label`| string | 64 characters | | Displayed text for the group.|
+|`tabs.groups.icons`| array | | | Displayed icons for the group. |
+|`tabs.groups.icons.size`| number | |✔️| Size of the icon in pixels, enumerated as `16`,`20`,`24`,`32`,`40`,`48`,`64`,`80`. Three image sizes are required: 16, 32, 80. |
+|`tabs.groups.icons.url`| URL| | | URL to the icon.|
+|`tabs.groups.icons.file`| string | 2048 characters| | Relative path to the file that contains the icon. This property is currently not supported as a relative path.|
+|`tabs.groups.controls`| array | | | Configures the buttons and menus in the group. |
+|`tabs.groups.controls.id`| string | 64 characters| ✔️ | Unique identifier for this control within the app. Must be different from any built-in control id in the Office application and any other custom control.|
+|`tabs.groups.controls.items`| object | | | Configures the items for a menu control. |
+|`tabs.groups.controls.items.id`| string | | ✔️ | Unique identifier for this item within the app. |
+|`tabs.groups.controls.items.type`| string enum | | ✔️ | Supported values: `menuItem`. |
+|`tabs.groups.controls.items.label`| string | 64 characters| ✔️ | Displayed text for the item. |
+|`tabs.groups.controls.items.icons`| array | | | Configures the icons for the custom item.|
+|`tabs.groups.controls.items.icons.size`| number | |✔️| Size of the icon in pixels, enumerated as `16`,`20`,`24`,`32`,`40`,`48`,`64`,`80`. Three image sizes are required: 16, 32, 80. |
+|`tabs.groups.controls.items.icons.url`| URL| | | URL to the icon.|
+|`tabs.groups.controls.items.icons.file`| string | 2048 characters| | Relative path to the file that contains the icon. This property is currently not supported as a relative path.|
+|`tabs.groups.controls.items.supertip`| | |✔️| Configures a supertip for the custom item.|
```

Comment:
@Rick-Kirkham what's a good one-line summary of "supertip"? Is there a doc I could link to for more info?

Reply:
@erikadoyle supertip: A brief, possibly multi-line, box of help information about a control in a UI that becomes visible when the cursor hovers over the control.

Reply:
Thanks! Added that.